src/Wallabag/ImportBundle/Import/PocketImport.php

```diff
@@ -252,16 +261,25 @@ private function parseEntries($entries)
                 );
             }
 
+            $pocketEntry['url'] = $url;
+            $pocketEntry['userId'] = $this->user->getId();
             $this->em->persist($entry);
             ++$this->importedEntries;
 
             // flush every 20 entries
             if (($i % 20) === 0) {
                 $this->em->flush();
             }
+
             ++$i;
         }
 
         $this->em->flush();
+
+        if ($this->rabbitMQ) {
+            foreach ($entries as $entry) {
+                $this->producer->publish(serialize($entry));
```

Comment:
Don't you need to send `$pocketEntry` instead of `$entry`?
Also, why do you need `$pocketEntry` by reference in the foreach? I don't get it...